packages/api-server/api_server/gateway.py

```diff
@@ -63,6 +62,20 @@
 )
 from .repositories import CachedFilesRepository
 
+# This conditional import is necessary for Alert related messages as they were
+# not released in to ROS 2 Jazzy. Once we stop supporting Jazzy, we can remove
+# this conditional import and use the Alert messages from rmf_task_msgs directly.
+# fmt: off
+# isort: off
+# pylint: disable=import-error,no-name-in-module
+if os.getenv("ROS_DISTRO") == "jazzy":
+    from rmf_alert_msgs.msg import Alert as RmfAlert, AlertResponse as RmfAlertResponse  # pyright: ignore[reportMissingImports,reportAttributeAccessIssue]
```

Comment:
For the jazzy branch, we cannot import messages not part of jazzy release. What I would recommend is to refactor out all the alert subscriptions to another file and create a jazzy version of that file that does not do anything. So we would have

`alert_gateway.py`
`alert_gateway_stub.py`

`alert_gateway_stub.py` would just be a stub that does not do anything. Then on the main gateway, do a conditional import for either one of them depending on the env. For pyright to work, we may also need to add a `if TYPE_CHECKING:` conditional import to tell it to use the stub. pyright may still complain errors in the jazzy CI so we may need to exclude the file as well (only in jazzy CI).